SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/CrushersDataBase.java

```diff
@@ -0,0 +1,156 @@
+package com.example.smartcarmqttapp;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import com.example.smartcarmqttapp.QuizContract.*;
+
+import androidx.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CrushersDataBase extends SQLiteOpenHelper {
+    private static final String DATABASE_NAME = "CrushersDataBase.db";
+    private static final int DATABASE_VERSION = 1;
+
+    private SQLiteDatabase db;
+
+
+    public CrushersDataBase(@Nullable Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        this.db = db;
+
+        final String CREATE_TABLE = "CREATE TABLE " + QuestionsTable.TABLE_NAME+
+            " (" + QuestionsTable._ID + " INTEGER PRIMARY KEY AUTOINCREMENT, " +
+            QuestionsTable.COLUMN_QUESTION + " TEXT," +
+            QuestionsTable.COLUMN_ANSWER1 + " TEXT, " +
+            QuestionsTable.COLUMN_ANSWER2 + " TEXT, " +
+            QuestionsTable.COLUMN_ANSWER3 + " TEXT, " +
+            QuestionsTable.COLUMN_ANSWER4 + " TEXT, " +
+            QuestionsTable.COLUMN_CORRECT_ANSWER + " INTEGER," +
+            QuestionsTable.COLUMN_EXPLANATION + " TEXT, " +
+            QuestionsTable.COLUMN_NEEDS_REVIEW + " INTEGER, " +
+            QuestionsTable.COLUMN_CATEGORY + " TEXT" + ")";
+    db.execSQL(CREATE_TABLE);
+    populateQuestionsTable();
+
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int pastVersion, int newVersion) {
+    db.execSQL("DROP TABLE IF EXISTS " + QuestionsTable.TABLE_NAME);
+    onCreate(db);
+    }
+
+    private List<Question> createQuestionsAddToList(){
+        //Place holder question values (3 topics(categories), 15 questions each)
+        return Arrays.asList(
+                //CATEGORY 1
+                new Question("placeholder question","answer1","answer2","answer3","answer4", 1, "explanation", 0, "cat1"),
```

Comment:
I know they are placeholders but the catergories for all questions are the same... is that intentional given the comments for separating the categories

Reply:
Yes, the comments are intended to separate the categories. I can change each question's category field to its respective category and push it again.